crates/ruff/src/jupyter/snapshots/ruff__jupyter__notebook__test__import_sorting.snap

```diff
@@ -0,0 +1,52 @@
+---
+source: crates/ruff/src/jupyter/notebook.rs
+---
+isort.ipynb:1:1: I001 [*] Import block is un-sorted or un-formatted
+  |
+1 | / from pathlib import Path
+2 | | import random
+3 | | import math
+4 | | from typing import Any
+  | |_^ I001
+5 |   import collections
+6 |   # Newline should be added here
+  |
+  = help: Organize imports
```

Comment:
Oh right. I'll need to update the printer to include cell number and use Jupyter index.